src/caret_analyze/infra/lttng/records_provider_lttng.py

```diff
@@ -1056,6 +1089,10 @@ def _compose_inter_proc_comm_records(
             columns.append(COLUMN_NAME.DDS_WRITE_TIMESTAMP)
         columns += [
             COLUMN_NAME.SOURCE_TIMESTAMP,
+        ]
+        if COLUMN_NAME.RMW_TAKE_TIMESTAMP in records.columns:
+            columns.append(COLUMN_NAME.RMW_TAKE_TIMESTAMP)
+        columns += [
```

Comment:
Is rmw_take necessary for inter_proc_comm?
If it is not strictly required, it would be better not to make changes in order to minimize modifications.
Also, if you decide to add the rmw_take column, please make sure to update the docstring.